File: python_programs/functions/fibonacci.py
'''
Print the fibonacci series
'''

def fibonacci(a,b,n=0):
    if(n==10):
        
        return
    c=a+b
    a=b
    b=c
    n+=1
    print(c,end=' ')
    fibonacci(a,b,n)

    
print(1,1,end=' ')
fibonacci(1,1)

File: c_programs/programs_2_23_2022/fibonacci.c
//WAP to print the fibonacci series

#include "stdio.h"

int main(){
    int limit,a,b,c;
    printf("Enter limit: ");
    scanf("%d",&limit);
    a=1;
    b=1;
    c=0;
    printf("%d %d",a,b);
    for (int i = 2; i < limit; i++)
    {
        c=a+b;
        a=b;
        b=c;
        printf(" %d",c);
    }
    
}